scripts/zones/Windurst_Walls/npcs/Arbitrix.lua

```diff
@@ -0,0 +1,54 @@
+-----------------------------------
+-- Area: Windurst Woods
+--  NPC: Arbitrix
+-- Gobbie Mystery Box
+-- !pos -215.5 0.0 -147.3
+-----------------------------------
+local ID = require("scripts/zones/Windurst_Walls/IDs")
+require("scripts/globals/settings")
+-----------------------------------
+
+function onTrigger(player,npc)
+    local playerAgeDays = (os.time() - player:getTimeCreated()) / 86400
+
+    local dailyTallyPoints = player:getCurrency("daily_tally")
+    local firstVisit = dailyTallyPoints == -1
+
+    local gobbieBoxUsed = player:getCharVar("gobbieBoxUsed")
+    local specialDialUsed = 1
+    local adoulinDialUsed = 1
+    local pictlogicaDialUsed = 1
+    local wantedDialUsed = 1
+
+    local adoulinOptionOff = 0x80
+    local pictlogicaOptionOff = 0x100
+    local wantedOptionOff = 0x1000
+    local hideOptionFlags = adoulinOptionOff + pictlogicaOptionOff + wantedOptionOff
+
+    if playerAgeDays >= MYSTERY_GOBBIE_BOX_MINIMUM_AGE_DAYS and firstVisit then
+        player:startEvent(519)
```

Comment:
Bonus points if you give the `startEvent` csids a variable instead in preparation of globalization~